nuxeo-management-rest-api/src/main/java/org/nuxeo/rest/management/work/manager/WorkManagerObject.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Nour Al Kotob
+ */
+package org.nuxeo.rest.management.work.manager;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.ws.rs.FormParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.apache.commons.lang3.StringUtils;
+import org.nuxeo.ecm.automation.AutomationService;
+import org.nuxeo.ecm.automation.OperationContext;
+import org.nuxeo.ecm.automation.OperationException;
+import org.nuxeo.ecm.automation.core.operations.services.workmanager.WorkManagerRunWorkInFailure;
+import org.nuxeo.ecm.core.api.Blob;
+import org.nuxeo.ecm.webengine.model.WebObject;
+import org.nuxeo.ecm.webengine.model.impl.DefaultObject;
+import org.nuxeo.runtime.api.Framework;
+
+/**
+ * @since 11.1
+ */
+@WebObject(type = "workmanager")
+public class WorkManagerObject extends DefaultObject {
+
+    public static final String TIMEOUT_SECONDS_PARAM_KEY = "timeoutSeconds";
+
+    /**
+     * Re-executes works in this DLQ for a repair purpose.
```

Comment:
"this DLQ" doesn't seem clear, maybe go for the same Javadoc as in the operation "Executes Works stored in the dead letter queue after failure."?

Reply:
"Executes Works stored in the dead letter queue after failure" it's ok for me just adding DLQ between brackets, because dlq is known in system like  ActiveMq...